scalafix/rules/src/main/scala/org/typelevel/fix/v0_5.scala

```diff
@@ -0,0 +1,14 @@
+package org.typelevel.fix
+
+import scalafix.v1._
+import scala.meta._
+
+class v0_5 extends SemanticRule("v0_5") {
+  override def fix(implicit doc: SemanticDocument): Patch =
+    doc.tree.collect {
+      case Importer(ref , _) if ref.toString == "io.github.davidgregory084" =>
+        Patch.replaceTree(ref, "org.typelevel.sbt")
+      case Importer(ref, _) if ref.toString == "_root_.io.github.davidgregory084" =>
+        Patch.replaceTree(ref, "_root_.org.typelevel.sbt")
```

Comment:
I wonder if it would be better to add another layer of package specificity here (`org.typelevel.sbt.tpolecat`) so that we can target future migrations better, and save the `org.typelevel.sbt` namespace for `sbt-typelevel`.